core/spec.md

```diff
@@ -80,12 +80,12 @@ to a "Schema" document - all within the same Registry instance:
 
 For easy reference, the JSON serialization of a Registry adheres to this form:
 
-``` text
+```yaml
 {
   "specVersion": "STRING",
   "id": "STRING",
   "name": "STRING", ?
-  "epoch": UINTEGER,
+  "epoch": "UINTEGER",
```

Comment:
Why the quotes when it's a number?

Reply:
Ok I got it. UINTEGERS are not quoted, like boolean. Will revert this particular change.